# Запуск этой программы вызовет панику, распечатает
# сообщение об ошибке и трейс выполнения и завершит
# работу с ненулевым статусом.
$ go run panic.go
panic: a problem

goroutine 1 [running]:
main.main()
	/.../panic.go:12 +0x47
...
exit status 2

# Обратите внимание, что в отличие от некоторых языков,
# которые используют исключения для обработки
# ошибок, в Go привычно использовать возвращающие
# значения, указывающие на ошибки.